src/styles/stars.scss

```diff
@@ -0,0 +1,51 @@
+$stars-count: 5;
```

Comment:
you have separate file for variables, please add this variable to the file

Reply:
I did. But, why? my point is: stars is aindependent block from product, and there is always 5 stars count. I can reuse this block anywhere and i want to configure it once at stars preferences, but variable.scss file describe product variables.